Library/Homebrew/utils/github.rb

```diff
@@ -312,7 +312,14 @@ def repository(user, repo)
   end
 
   def search_code(**qualifiers)
-    search("code", **qualifiers)
+    matches = search("code", **qualifiers)
+    return matches if matches.blank?
+
+    matches.map do |match|
+      # .sub workaround for GitHub returning preceding /
+      match["path"] = match["path"].sub(%r{^/}, "")
```

Comment:
```suggestion
      match["path"] = match["path"].delete_prefix("/")
```

Reply:
@reitermarkus Good call. Will open a follow-up PR with this in a sec, merging as-is to unblock CI on other PRs.

Reply:
@reitermarkus https://github.com/Homebrew/brew/pull/10057, thanks!